packages/provider-utils/src/test/unified-test-server.ts

```diff
@@ -58,10 +58,30 @@ export type UrlHandlers<
 class TestServerCall {
   constructor(private request: Request) {}
 
-  get requestBody() {
+  get requestBodyJson() {
     return this.request!.text().then(JSON.parse);
   }
 
+  get requestBodyMultipart() {
+    const requestClone = this.request.clone();
+
+    return requestClone.text().then(text => {
```

Comment:
why is this needed before the check? headers should be available

Reply:
Sorry, artifact of refactoring